src/App.css

```diff
@@ -1,5 +1,9 @@
 @import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500;600;700&display=swap');
```

Comment:
Better to load fonts in index.html as you can preconnect and load the fonts before user sees anything.

Reply:
Great suggestion, thanks!